S21/rkarmuri/ConvertToLowerCase.java

```diff
@@ -0,0 +1,98 @@
+                replace('Α', 'α').replace('Β', 'β').
+                replace('Γ', 'γ').replace('Π', 'π').
+                replace('Ό', 'ό').replace('Λ', 'λ').
+                replace('Η', 'η').replace('Δ', 'δ').
+                replace('Ζ', 'ζ').replace('Θ', 'θ').
+                replace('Μ', 'μ').replace('Ν', 'ν').
+                replace('Ξ', 'ξ').replace('Ρ', 'ρ').
+                replace('Ω', 'ω').replace('Ψ', 'ψ').
+                replace('Χ', 'χ').replace('Φ', 'φ').
+                replace('Υ', 'υ').replace('Τ', 'τ');
+        String outputString;//Variable that is used to return the desired lowercase to test code method
+        switch (language) {
+            //This case is to convert English, English(US), English(Ireland), English(Latin) and Spanish words
+            case "en":
+            case "en-US":
+            case "en-IE":
+            case "en-Latn":
+            case "es":
+                System.out.println("String after converting to lower case: " + wordLowerCase);
+                outputString = wordLowerCase;
+                break;
+            //This case is to convert Turkish and Azerbaijan words
+            case "tr":
+            case "az":
+                System.out.println("String after converting to lower case: " + trazString);
+                outputString = trazString;
+                break;
+            //This case is to convert all Greek words
+            case "el":
+                if (wordSample.endsWith("Σ")) {
+                    //Replacing Word ending with Sigma condition
+                    String newString = wordLowerCase.replace('Σ', 'ς');
+                    System.out.println("String after converting to lower case: " + newString);
+                    outputString = newString;
+                } else {
+                    //Print and return the output lowercase in greek
+                    System.out.println("String after converting to lower case:" + greekString);
+                    outputString = greekString;
+                }
+                break;
+            //This case is to convert Russian words
+            case "ru-Latn":
+                System.out.println("String after converting to lower case: " + russString);
+                outputString = russString;
+                break;
+            //This case is to convert Hungarian words
+            case "hu":
+                System.out.println("String after converting to lower case: " + hungarianString);
+                outputString = hungarianString;
+                break;
+            //This case is to convert all Irish words
+            case "ga":
+            case "ga-IE":
+                char c = wordSample.charAt(1);
+                if (wordSample.startsWith("n") || wordSample.startsWith("t")) {
+                    if (c == 'A' || c == 'E' || c == 'I' || c == 'O' || c == 'U' || c == 'Á' || c == 'É' || c == 'Í' || c == 'Ó' || c == 'Ú') {
```

Comment:
Can we use the replace() function here or just combine these replacements to the array and then check if the c can be replaced?

We can do it like this
char[] replacementLettersIrish = {'', '', ''}
for (int i = 0; i < letters.length; i++){
               if (c == letters[i]){
                    String IrishString1 = wordLowerCase.charAt(0) + "-" + wordLowerCase.substring(1);
               }
}